recipes/orange3/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "3.3.6" %}
+
+package:
+  name: orange3
```

Comment:
orange3 is the same name that is used for this package in default repository. I can change it if needed.